Controllers/PizzaController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using PizzaOrdering_Mvc.DTO;
+using PizzaOrdering_Mvc.Models;
+using PizzaOrdering_Mvc.PizzaApp_DbContext;
+
+namespace PizzaOrdering_Mvc.Controllers
+{
+    public class PizzaController : Controller
+    {
+        private readonly PizzaAppDbContext _pizzaLoading;
+        public PizzaController(PizzaAppDbContext pizzaLoading)
+        {
+            _pizzaLoading = pizzaLoading;
+        }
+        [HttpGet]
+        public IActionResult CreatePizza()
+        {
+            try
+            {
+                ViewBag.bases = _pizzaLoading.Base.ToList();
+                ViewBag.toppings = _pizzaLoading.Toppings.ToList();
+                ViewBag.sizes = _pizzaLoading.Size.ToList();
+                var pizzaDetails = new PizzaViewModel();
+
+                return View(pizzaDetails);
+            }
+            catch (Exception e)
+            {
+                return View("Home", "Index");
```

Comment:
In case of exception, we need to show some error message on the frontend, and same View should be returned here.